app_basics/commands/validators.py

```diff
@@ -0,0 +1,36 @@
+from typing import Tuple
+# import .constants as con
+from .constants import constants as con
+
+
+# raw_cmd = '//quit' # input from user
+# // EXIT_CMD indexing
+def is_command_valid(raw_cmd: str) -> Tuple[str, bool]:
+    """
+    slug - the category the cmd relates to.
+    """
+    for slug, cmd in con.items():
+        if type(cmd) is not tuple:
+            print("The constants value must be in tuple format !!!")
+            return False
```

Comment:
is `False` accurate here? What does it mean when `cmd` is not a tuple? is it the fault of the developer of the user?
What can you do here to make the code stop and app crash to notify the developer that the cmd mapping is invalid?

Reply:
Make sure to use **the same return _type_** in each `return`